test/unit/lib/gpHelper.js

```diff
@@ -98,4 +98,42 @@ describe('gp counts helper', () => {
       expect(message[0]).to.equal('1 GP');
     });
   });
+
+  describe('getPersonSingular', () => {
+    it('should return \'are\' for more than one male and female gp', () => {
```

Comment:
I'd be tempted to have this test have both GP counts as 2 and reword it to be `should return \'are\' when there are several male GPs and several female GPs`